src/pmdas/bpf/modules/GNUmakefile

```diff
@@ -0,0 +1,54 @@
+TOPDIR = ../../../..
+include $(TOPDIR)/src/include/builddefs
+
+IAM	= bpf
+MODULES	= biolatency.so runqlat.so
+BPF_MODULES = biolatency.bpf.o runqlat.bpf.o 
+MODULETMP = $(PCP_PMDAS_DIR)/$(IAM)/modules
+MODULEDIR = $(PCP_PMDASADM_DIR)/$(IAM)/modules
+
+$(eval $(shell grep ^VERSION_ID= /etc/os-release))
+$(eval $(shell grep ^ID= /etc/os-release))
+
+# Use the clang pipeline to emit LLVM to LLD and emit BPF straight to an ELF .o.
+# The GCC pipeline has parts of this but not completely, and in any case, likely
+# requires version/feature detection too.
+#
+# Some distributions place commands at different names (llvm-dis vs llvm-dis-12),
+# so detect distro directly.
+#
+# clang-12 special notes: https://www.spinics.net/lists/netdev/msg689901.html
+#     nb: opt and llvm-dis appear as opt-12 and llvm-dis-12 on some OSs
+
+.PHONY: .FORCE
+FORCE:
+
+ifeq "$(ID)$(VERSION_ID)" "fedora34"
```

Comment:
Instead of these distribution-specific checks (maintenance nightmare) how about checking for the different clang versions in configure.ac?  (and passing the correct flags for each down from that top level)

Reply:
Agreed, I'm a bit more comfortable with the different OS environments so I think I can make this work now.

Reply:
fixed in f6d963a5c